### PR TITLE
2018 End of Year Survey

### PR DESCRIPTION
**Pull request by kz**
To review: [2018-07-18-member-survey.pdf](https://github.com/icdocsoc/emails/files/2194978/2018-07-18-member-survey.pdf)

### CODE REVIEW

File: xml/2018-07-15-end-of-year-survey.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="2018 Member Survey"
+  agenda="false"
+  social="false"
+  logos="false">
```
Comment (nvgrw) [inline xml/2018-07-15-end-of-year-survey.xml]: Perhaps we should include social and logos? We tend to only exclude them on mail shots etc. Default is true so you can just remove these attributes.

File: xml/2018-07-15-end-of-year-survey.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    all of us, we’d like to collect some data about our performance this year.
+
+    We're also interested in finding out how your experience has been so far
+    with DoCSoc so that we can continue to improve our society.
+
+    The survey is ANONYMOUS. If you wish to be eligible for the prize, we will
+    ask for your username. However, we will NEVER use your username in
+    conjunction with any of the other answers from this survey.
+
+    Complete the survey here: [https://docsoc.co.uk/survey/](https://docsoc.co.uk/survey/)
```
Comment (nvgrw) [inline xml/2018-07-15-end-of-year-survey.xml]: The preprocessor does link detection so you shouldn't need to write the link explicitly if you want it to show up with the same text as the destination.

**Comment by nvgrw** [comment]
looks good o me

**Comment by kz** [comment]
Awesome. Will send this tomorrow.

**Pull request merged**